third_party/brotli/.gitattributes

```diff
@@ -1,54 +0,0 @@
-tests/testdata/* binary
```

Comment:
Is it fine to remove this?

Reply:
You are right, better safe than sorry here. I'll put that back